js/languagePickerNavView.js

```diff
@@ -8,7 +8,16 @@ define([
         
         tagName: 'button',
         
-        className: 'languagepicker-icon base icon icon-language',
+        className: function () {
+            var classNames = 'languagepicker-icon base icon';
+            var customClass = 'icon-language-2';
+
+            if (this.model.has('_languagePickerIconClass') && this.model.get('_languagePickerIconClass') !== '') {
+                customClass = this.model.get('_languagePickerIconClass');
+            }
```

Comment:
Rather than doing a `has()` then a `get()` then an identical `get()`, we could just have one function call:
```js
var customClass = this.model.get('_languagePickerIconClass') || 'icon-language-2';
```
This checks for existence and empty string in one go. Don't feel obliged to change though.